index.js

```diff
@@ -67,7 +102,15 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+function countByRating(movies) {
+  let movieRatings = {};
+  
+  for (let i = 0; i < movies.length; i++) {
+    let ratingCount = movies[i].rated
+    movieRatings[ratingCount] = (movieRatings[ratingCount] + 1) || 1
```

Comment:
Great short circuit - what does the left hand side of our ` || ` return in the case that there is `!movieRatings[ratingCount]+1`